.github/workflows/alpha.yaml

```diff
@@ -0,0 +1,23 @@
+name: alpha
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

Comment:
Any special reason not to go for v1.18 ?

Reply:
Or 1.19

Reply:
I will bump to 1.19